Probesys/bileto
# imp: Check permissions

Related to https://github.com/Probesys/bileto/issues/88

Changes proposed in this pull request:

- Verify (admin and orga) permissions in controllers
- Hide interface elements based on current user's permissions
- List only tickets that user has access to

How to test the feature manually:

1. Create different roles with different permissions (e.g. read-only, user, technical)
2. Use the interface, try to access forbidden endpoints
3. Check all is coherent and that you don't face unexpected errors

Pull request checklist:


_If you think one of the item isn’t applicable to the PR, please check it
anyway and precise `N/A` next to it._
